src/main/scripts/src/app/settings/organization/organization.component.ts

```diff
@@ -53,20 +53,33 @@ export class OrganizationComponent implements OnInit {
 
   public onSave() {
     if (this.orgCode) {
-      this.organizationService.editOrganization(this.orgCode, this.organization);
+      this.organizationService.editOrganization(this.orgCode, this.organization)
+        .subscribe(response => {
+          if (response.ok) {
+            this.location.back();
```

Comment:
There are 2 ways of creating services which fetch data from server:

1. Inside observable create function which will fetch data, subscribes and assignes data to variable. This is prefect for example for data which doesn't need to be refreshed that much and can be shared amongst other components
2. Create observable and subscribe to it in components. This is prefered when data should not be shared in components and whenever component is presented and rendered it will fetch newest data from server.

First solution has advantage of sharing data, but the data might be outdated (we have to refresh the data). Second solution avoids problem with refreshing data when rendering, but if we'd need to refresh data which are already fetched in component there is no way of triggering this refresh.